.gitbook.yaml

```diff
@@ -199,3 +199,201 @@ redirects:
+  smart-contracts/filecoin-evm-runtime/filforwader: smart-contracts/filecoin-evm-runtime/filforwarder.md
+  smart-contracts/wallets/metamask: basics/assets/metamask-setup.md
+  software: storage-providers/architecture/lotus-components.md
+  storage-provider/basics/how-providing-storage-works: basics/what-is-filecoin/storage-model.md
+  storage-provider/hardware-requirements: storage-providers/skills/storage.md
+  storage-provider/hardware/architectures: storage-providers/infrastructure/reference-architectures.md
+  storage-provider/hardware/hardware-requirements: storage-providers/infrastructure/reference-architectures.md
+  storage-provider/how-providing-works: basics/what-is-filecoin/storage-model.md
+  storage-provider/rewards: storage-providers/filecoin-economics/block-rewards.md
+  storage-provider/rewards/storage-provider-rewards: storage-providers/filecoin-economics/block-rewards.md
+  storage-provider/storage-provider-rewards: storage-providers/filecoin-economics/block-rewards.md
+  storage-provider/videos/actors: basics/the-blockchain/actors.md
+  storage-provider/videos/cryptoecon: basics/what-is-filecoin.md/crypto-economics
+  storage-provider/videos/fvm: smart-contracts/fundamentals.md
+  storage-provider/videos/gdpr: storage-providers/skills/linux.md
+  storage-provider/videos/green: basics/what-is-filecoin.md
+  storage-provider/videos/hardware: storage-providers/infrastructure/reference-architectures.md
+  storage-provider/videos/implementations: nodes/implementations.md
+  storage-provider/videos/opendata: basics/what-is-filecoin.md
+  storage-provider/videos/pl: basics/project-and-community/ways-to-contribute.md
+  storage-provider/videos/rclone: reference/general.md
+  storage-provider/videos/roadmap2022: basics/project-and-community/the-filecoin-project.md
+  storage-provider/videos/seagate: basics/what-is-filecoin.md
+  storage-provider/videos/supermicro: basics/what-is-filecoin.md
+  storage-provider/videos/usecase: basics/what-is-filecoin.md
+  storage-provider/videos/vc: basics/what-is-filecoin.md
+  storage-provider/videos/virtual: basics/what-is-filecoin.md
+  storage-provider/videos/why: basics/what-is-filecoin.md
+  store/estuary: storage-providers/basics/quickstart-guide.md
+  store/lotus: nodes/implementations/lotus.md
+  store/lotus/retrieve-data: basics/how-retrieval-works.md
+  store/lotus/store-data: basics/what-is-filecoin/storage-model.md
+  store/lotus/very-large-files: basics/how-storage-works/filecoin-and-ipfs.md
+  store/overview: basics/what-is-filecoin/storage-model.md
+  store/slate: basics/what-is-filecoin.md
+  store/starling: basics/what-is-filecoin.md
+  store/tools/estuary: reference/general.md
+  store/tools/starling: basics/project-and-community/filecoin-faqs.md
+  uild/examples/sample-architectures: storage-providers/infrastructure/reference-architectures.md
+  uild/lotus/api-token-generation: reference/built-in-actors/protocol-api.md
```

Comment:
Just wanna confirm that these are meant to start with `uild/...` and aren't typos. Are these two lines correct @SgtCoin ?

Reply:
Fixed it. It was a typo, good catch.

Reply:
Danke!